### PR TITLE
Fix most module cyclic references

### PR DESCRIPTION
**Pull request by hshristov**
There is one left for ios and android:

- http -> http-request -> image-source
that can't be fixed easily and will be removed once we delete image-source module because it is obsolete anyway.

And one more for android only:
- frame -> fragment 
that could be removed if we use global object.

Updated madge version

### CODE REVIEW

**Comment by hshristov** [comment]
run ci

**Comment by hshristov** [comment]
run ci

**Comment by hshristov** [comment]
uitests

**Comment by hshristov** [comment]
uitests-ios-pickers

File: tns-core-modules/ui/frame/frame-common.ts [hunk 1]
```diff
@@ -7,34 +7,10 @@ import { View, CustomLayoutView, isIOS, isAndroid, traceEnabled, traceWrite, tra
...
 import { parse, loadPage } from "../builder";
-import * as application from "../../application";
 import { profile } from "../../profiling";
 
-export { application };
-
 export * from "../core/view";
 
-function onLivesync(args: EventData): void {
-    // give time to allow fileNameResolver & css to reload.
```
Comment (PanayotCankov) [inline tns-core-modules/ui/frame/frame-common.ts]: Mind the setTimeout

**Comment by hshristov** [comment]
ios-pickers

**Comment by hshristov** [comment]
ios-issues

**Comment by SvetoslavTsenov** [comment]
uitests-ios-issues

**Comment by SvetoslavTsenov** [comment]
ios-issues

**Pull request merged**